open-api/rest-catalog-open-api.yaml

```diff
@@ -1581,6 +1607,17 @@ components:
         type: string
       example: [ "accounting", "tax" ]
 
+    PageToken:
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service
+        that they would prefer the response to be paginated.
+
+        Clients will initiate the request by sending an empty pageToken e.g. GET /tables?pageToken or /tables?pageToken=
+        For servers that support pagination, they will recognize pageToken and return a next pageToken in response if there are more results available.
+        After initial request, it is expected that the next pageToken from the last response be used in the subsequent request.
+        For servers that do not support pagination, they will ignore the pageToken and return all results.
```

Comment:
```suggestion
        Servers that do not support pagination will ignore next-page-token and return all results.
```

Reply:
Actually I think the current sentence reads fine and is for the initial case of sending the pageToken.

Reply:
The main idea here was to avoid using `they` to refer to servers, that's why I suggested an alternative wording that is easier to read and avoids using `they`

Reply:
Will fix this.